user_manual/docs/firmware.md

```diff
@@ -5,18 +5,39 @@ Updating Firmware
 
 # Overview {#update-overview}
 
-Discuss boot loader vs application firmware.
+The **Firmware** is the code that runs on the board.
+
+The **Bootloader** is a mode that allows the firmware to be written to the board
 
 # Application Firmware Update {#update-steps}
 
 ## Enter Bootloader Mode {#update-bootloader}
+To enter the Bootloader mode hold the button closest to the usb port when powering up. If you are successful the top LED will be solid and the bottom three LEDS will flash together
 
 ## Flash Firmware {#update-flash}
+Download the latest firmware package from [firmware] and note its location
+### On Linux {#On-Linux}
```

Comment:
I think we want to make these references unique, so #update-flash-linux is probably a better bet.